frontend/src/features/vessel_sidebar/reporting/current/CreateOrEditReporting.tsx

```diff
@@ -14,12 +16,19 @@ export function CreateOrEditReporting() {
   const { selectedVesselIdentity } = useAppSelector(state => state.vessel)
   const editedReporting = useAppSelector(state => state.reporting.editedReporting)
   const [newReportingFormIsOpen, setNewReportingFormIsOpen] = useState(false)
+  const previousSelectedVesselIdentity = usePrevious(selectedVesselIdentity)
 
   const close = useCallback(() => {
     setNewReportingFormIsOpen(false)
     dispatch(setEditedReporting(null))
   }, [dispatch])
 
+  useEffect(() => {
```

Comment:
Same thing: can be done in Redux.

Reply:
We'll need to modify too much use-cases, if we move that to Redux